utils/release/operator/make_bundles.sh

```
@@ -31,6 +31,11 @@ cd $env
 dns_operator_version=$(mod_version $DNS_OPERATOR_VERSION)
 dns_operator_image="quay.io/kuadrant/dns-operator:$dns_operator_version"
 
+# Set desired mcp-gateway images
+mcp_gateway_version=$(mod_version $MCP_GATEWAY_VERSION)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)